### ISSUE TITLE
Small documentation inconsistency

### ISSUE THREAD

**Issue by CedricPerottet** [keras-team/keras-tuner#387]
In the [Subclassing Tuner for Custom Training Loops](https://github.com/keras-team/keras-tuner/blob/master/docs/templates/tutorials/subclass-tuner.md) documentation, there is an inconsistency :

> 
> ### Overriding `run_trial`.
> 
> There are two ways to write `run_trial`. One is to leverage `Tuner`'s built-in callback hooks, which send the value of the `objective` to the `Oracle` and save the latest state of the Model. These hooks are:
> 
> * `self.on_epoch_end`: Must be called. Reports results to the `Oracle` and saves the Model. The `logs` dictionary passed to this method must contain the **`objective`** name.

The document states that the 'objective' key must be present in the logs dictionnary, but in all the examples, only the 'loss' key is present  :

> ```python
> class MyTuner(kt.Tuner):
> 
>     def run_trial(self, trial, ...):
>         model = self.hypermodel.build(trial.hyperparameters)
>         for epoch in range(10):
>               epoch_loss = ...
>               self.on_epoch_end(trial, model, epoch, logs={'loss': epoch_loss})
> ```

Regards,
Cédric
